src/users/users.controller.ts

```diff
@@ -262,4 +282,20 @@ export class UsersController {
     await this.usersService.unfollowArtist(req.user.email, params.followeeName);
     return { message: 'success to unfollow user', success: true };
   }
+
+  @ApiBearerAuth()
```

Comment:
This decorator is unnecessary since I declared it in global.

Reply:
Can I delete all of its?